compiler/rustc_middle/src/ty/print/pretty.rs

```diff
@@ -946,7 +945,10 @@ pub trait PrettyPrinter<'tcx>:
         &mut self,
         trait_ref: ty::PolyTraitRef<'tcx>,
         proj_ty: Option<(DefId, ty::Binder<'tcx, Ty<'tcx>>)>,
-        traits: &mut FxHashMap<ty::PolyTraitRef<'tcx>, BTreeMap<DefId, ty::Binder<'tcx, Ty<'tcx>>>>,
+        traits: &mut FxHashMap<
+            ty::PolyTraitRef<'tcx>,
+            FxHashMap<DefId, ty::Binder<'tcx, Ty<'tcx>>>,
```

Comment:
Same question?

Reply:
Used in the same calls as above https://github.com/rust-lang/rust/pull/90749#discussion_r779062831.